ingestion/src/metadata/utils/connections.py

```diff
@@ -198,6 +201,14 @@ def _(connection: GlueConnection, verbose: bool = False) -> GlueClient:
     return glue_connection
 
 
+@get_connection.register
+def _(connection: GlueConnectionPipeline, verbose: bool = False) -> GlueClient:
```

Comment:
one question I would have here after decoupling Glue db and pipeline ingestion is regarding the `test_connection`. The current connectivity test for both is centralised by listing the workflows. That would not be needed anymore for ingesting dbs, so maybe we should have two tests. Otherwise, we might get false negatives depending on the permissions assigned to the role